tests/test_docx.py

```diff
@@ -45,8 +45,6 @@ class ConvertDocxToHtmlTestCase(TestCase):
         # level upperRomans are converted.
         'external_image',
         'export_from_googledocs',
-        'fake_subscript',
```

Comment:
I don't think they are difficult to manage. As long as their names describe what should be testing (which for the most part is done), then all you have to do is open the file to see make sure it actually doing what we think it's doing. I am still quite against removing these files because we are testing a real life document to make sure our implementation works on a real document. Not just he XML we create (that may or may not actually be correct).